app/controllers/organizations_controller.rb

```diff
@@ -14,11 +14,12 @@ def edit
 
   def update
     @organization = current_organization
+
     if OrganizationUpdateService.update(@organization, organization_params)
       redirect_to organization_path(@organization), notice: "Updated your organization!"
     else
-      flash[:error] = "Failed to update your organization."
-      render :edit
+      flash[:error] = @organization.errors.full_messages.first
+      redirect_to edit_organization_path(@organization)
```

Comment:
Is there a reason this changed? If you redirect, you lose all the form information that has been entered. It's better to keep `render :edit`.